encord/utilities/benchmark_utilities.py

```diff
@@ -0,0 +1,210 @@
+from typing import Dict, List
+
+from shapely.geometry import Polygon, box
+
+from encord.client import EncordClientProject
+
+
+def get_project_labels_consensus(
+    baseline_project: EncordClientProject,
+    comparing_projects: List[EncordClientProject],
+    label_types: List[str],
+    threshold: float = 0.7,
+) -> Dict[str, Dict[str, float]]:
+    """
+    Calculate performance metrics of a label generating agent when compared with consensus reached by other agents.
+
+    Args:
+        baseline_project:
+            The project to be evaluated.
+        comparing_projects:
+            The list of projects where consensus is extracted.
+        label_types:
+            The list of feature node hashes indicating objects from baseline_project to be evaluated.
+        threshold:
+            Used in objects comparison. Two objects' instances are considered the same object if their Jaccard
+            similarity coefficient is greater or equal than the threshold; otherwise, they represent distinct objects.
+
+    Returns:
+        Consensus score (precision and recall) for objects from baseline_project indexed by their feature node hashes.
+
+    Raises:
```

Comment:
I'm thinking if we could expect various KeyErrors if any of the labels is mis-formed. Something to think about, especially as a user might expect nothing else to throw here given how good the function documentation is.